cloud/jenkins/psmdb_operator_minikube.groovy

```diff
@@ -226,11 +226,22 @@ pipeline {
                     runTest('init-deploy')
                     runTest('limits')
                     runTest('scaling')
+                    runTest('monitoring')
```

Comment:
Please do not add these tests for minikube. These tests do not work on minikube @cap1984  handle it.